tests/test_standard_tags.py

```diff
@@ -161,23 +161,64 @@ def test_route(self):
 
 
 @released(
-    dotnet="2.13.0",
-    golang="1.39.0",
-    java="0.107.1",
-    nodejs="3.2.0",
-    php="0.76.0",
-    python=PYTHON_RELEASE_GA_1_1,
+    dotnet="?",
+    golang="1.44.0",
+    java="?",
+    nodejs="?",
+    php="?",
+    python="?",
     ruby="?",
 )
 @coverage.basic
 class Test_StandardTagsClientIp(BaseTestCase):
     """Tests to verify that libraries annotate spans with correct http.client_ip tags"""
 
     def test_client_ip(self):
-        headers = {"X-Cluster-Client-IP": "10.42.42.42, 43.43.43.43, fe80::1"}
+        """Test http.client_ip is reported in the default scenario which has ASM enabled"""
```

Comment:
for the clarety of the test, I would make three methods : 

* test_client_ip:, check that client IP is reported on an attack, and on a normal request
* test_tags_on_attack:  check that `appsec.event` ... are in meta on an attack
* test_tags_on_normal_request:  check that `appsec.event` ... are NOT in meta on an attack

I'll be eaiser for non-pythonic user to understand, rather that a function that returns a function. I can make this change if it helps